nvtabular/loader/backend.py

```diff
@@ -480,16 +499,17 @@ def _create_tensors(self, gdf):
             gdf.drop(columns=column_names, inplace=True)
 
             scalars, lists = self._separate_list_columns(gdf_i)
+
             x = None
             if scalars:
+                # should always return dict column_name: values, offsets (optional)
```

Comment:
Is there a reasonable way to enforce that it does return this?

Reply:
Do you mean like an assert? i.e. `assert isinstance(x, dict)` ?

Reply:
Yeah, maybe something like that? Not sure if we really need one here, just trying to think about how to turn the comment into code.